projects/plugins/jetpack/extensions/blocks/recurring-payments/edit.jsx

```diff
@@ -15,17 +15,41 @@ import { getBlockStyles } from './util';
 const BLOCK_NAME = 'recurring-payments';
 
 export default function Edit( { attributes, clientId, setAttributes } ) {
-	const { align, planId, width } = attributes;
-	const planIds = useMemo( () => {
-		const _planIds = ( '' + planId ).split( '+' ).map( id => parseInt( id, 10 ) );
-		return _planIds;
-	}, [ planId ] );
+	const { align, planId, planIds, width } = attributes;
+
+	const _planIds = useMemo( () => {
+		return planIds || ( planId ? [ planId ] : [] );
+	}, [ planId, planIds ] );
+
+	// planId is a integer, planIds is a plus-separated string of integers.
```

Comment:
planIds itself is an array, the plus-separated string is only in the URL and the uniqueId.
```suggestion
	// planId is a integer, planIds is an array of integers.
```